ec/resources/waf/managed_rule.go

```diff
@@ -0,0 +1,480 @@
+						"process_request_body": {
+							Type:        schema.TypeBool,
+							Description: "Determines whether JSON payloads will be inspected.",
+							Optional:    true,
+						},
+						"response_header_name": {
+							Type:        schema.TypeString,
+							Description: "Determines the name of the response header that will be included with blocked requests.",
+							Optional:    true,
+						},
+						"total_arg_length": {
+							Type:        schema.TypeInt,
+							Description: "Indicates the maximum number of characters for the query string value.",
+							Required:    true,
+						},
+						"validate_utf8_encoding": {
+							Type:        schema.TypeBool,
+							Description: "Indicates whether WAF may check whether a request variable (e.g., ARGS, ARGS_NAMES, and REQUEST_FILENAME) is a valid UTF-8 string.",
+							Optional:    true,
+						},
+						"xml_parser": {
+							Type:        schema.TypeBool,
+							Description: "Determines whether XML payloads will be inspected.",
+							Optional:    true,
+						},
+					},
+				},
+				Description: "Contains settings that define the profile for a valid request.",
+			},
+			"policies": {
+				Type:        schema.TypeList,
+				Description: "Contains a list of policies that have been enabled on this managed rule.",
+				Optional:    true,
+				Elem:        &schema.Schema{Type: schema.TypeString},
+			},
+			"rule_target_update": {
+				Type:     schema.TypeSet,
+				Optional: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"is_negated": {
+							Type:        schema.TypeBool,
+							Description: "Determines whether the current target, as defined within this object, will be ignored when identifying threats.",
+							Optional:    true,
+						},
+						"is_regex": {
+							Type:        schema.TypeBool,
+							Description: "Identifies a rule that will be disabled by its system-defined ID.",
+							Required:    true,
+						},
+						"replace_target": {
+							Type:        schema.TypeString,
+							Description: "Defines the data source (e.g., REQUEST_COOKIES, ARGS, GEO, etc.) that will be used instead of the one defined in the target parameter.",
+							Optional:    true,
+						},
+						"rule_id": {
+							Type:        schema.TypeString,
+							Description: "Identifies a rule by its system-defined ID.",
+							Required:    true,
+						},
+						"target": {
+							Type:        schema.TypeString,
+							Description: "Identifies the type of data source (e.g., REQUEST_COOKIES, ARGS, GEO, etc.) for which a target will be created.",
+							Required:    true,
+						},
+						"target_match": {
+							Type:        schema.TypeString,
+							Description: "Identifies a name or category (e.g., cookie name, query string name, country code, etc.) for the data source defined in the target parameter.",
+							Required:    true,
+						},
+					},
+				},
+				Description: "This array identifies each rule that has been disabled using these properties",
```

Comment:
Is this the correct description? Sounds like it's for disabled_rules

Reply:
@stevenpaz great catch, thank you. corrected.